test/models/FailedLookups.spec.ts

```diff
@@ -39,6 +39,7 @@ describe('Failed Lookups Model', () => {
   describe('Indexes', () => {
     it('should use index when find by osdbHash', async() => {
       await FailedLookupsModel.create({ osdbHash: '8e245d9679d31e12' });
+      await FailedLookupsModel.create({ osdbHash: '8e245d9679d31e13' });
```

Comment:
```suggestion
      await FailedLookupsModel.init();
      await FailedLookupsModel.create({ osdbHash: '8e245d9679d31e12' });
```

The above Promise fufills when the index build is done (which is in the background by default) so removes the race condition from the test